src/api/cpp/cvc5.h

```diff
@@ -433,7 +434,10 @@ class CVC5_EXPORT Sort
   bool isDatatype() const;
 
   /**
-   * Is this a parametric datatype sort?
+   * Is this a parametric datatype sort? A parametric datatype sort is either
+   * one that is returned by a call to mkDatatypeSort or mkDatatypeSorts for a
```

Comment:
```suggestion
   * one that is returned by a call to Solver::mkDatatypeSort() or Solver::mkDatatypeSorts() for a
```
Note: you will have to reformat this after applying these changes